resources/views/lms/components/dashboard-sidebar.blade.php

```diff
@@ -20,12 +20,13 @@
             <div class="user-info">
                 <p>
                     <span class="username">
-                        {{ auth()->user()->name }}
+                        {{ auth()->user()->name }}<br>
+                         {{ '@' .  auth()->user()->username }}
```

Comment:
Se for utilizar isso em mais partes do sistema, pode ser interessante ter um mutator para já trazer formatado

Reply:
Vamos ver nos próximos PR's se é interessante e caso for, lança uma chore pra gente =)